### ISSUE TITLE
COERCE doesn't coerce to STRING

### ISSUE THREAD

**Issue by stassats** [drmeister/clasp#59]
(coerce '(#\a #\b) 'string)
=>
#( #\a #\b )

**Comment by Shinmera** [comment]
Can confirm --albeit differently-- on b05884f.
```
> (coerce '(#\a #\b) 'string)

#<vector 0x7f79fdb9b351>
```

**Comment by Shinmera** [comment]
Fixed on dea135513350b3509bc837edf4ca34bcf530158a
```
> (coerce '(#\a #\b) 'string)

"ab"
```
